### ISSUE TITLE
Add optional pin/fingerprint protection

### ISSUE THREAD

**Issue by derekantrican** [BoostIO/boostnote-mobile#72]
Simplenote (and I'm sure - others) has this feature. Whenever the app is opened (or device is unlocked while app is open, or app is switched to from multitasking) it will require a pin (or, optionally, a fingerprint) in order to access the notes. Based on the user's preference, obviously

**Comment by kohei-takata** [comment]
Thanks @derekantrican !
So it for security?
I'm not sure how many people want this feature.
So I will not develop this feature sooner.
PRs are welcome!

**Comment by derekantrican** [comment]
Yeah, for security. Of course - it's definitely a "down-the-line" feature
